CONTRIBUTING.md

```diff
@@ -1,11 +1,13 @@
 # Contributing
 
-We welcome any code contributions and hope that this guide facilitates
-an understanding of the PyFluent code repository. It is important to
-note that while the PyFluent software package is maintained by Ansys and any
-submissions are reviewed thoroughly before merging, we still seek to foster
-a community that can support user questions and develop new features to make
-this software a useful tool for all users. As such, we welcome and encourage
-any questions or submissions to this repository.
+The PyFluent project welcomes contributions from the community. Please see the
+[README](https://github.com/pyansys/pyfluent/blob/main/README.rst) to better
+understand the code repository. It is important to note that while the PyFluent
```

Comment:
```suggestion
The PyFluent project welcomes contributions from the community. To better
understand the code repository, see the [README](https://github.com/pyansys/pyfluent/blob/main/README.rst).
While the PyFluent
```
Previously, "PyFluent project" was used. In this section "PyFluent library package" is used, and both terms aren't necessary. To avoid confusion, I've been using just the library name, such as "PyFluent", to be the noun (rather than an adjective).